pattern_import_export_csv/migrations/12.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,27 @@
+#  Copyright (c) Akretion 2020
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+from openupgradelib import openupgrade
+
+FIELDS_DUMB_COPY = ["export_format", "csv_value_delimiter", "csv_quote_character"]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    def migrate_ir_exports_misc_fields():
+        cpy_fragment = ",".join(
+            [
+                "{field} = ir_exports.{field}".format(field=field)
+                for field in FIELDS_DUMB_COPY
+            ]
+        )
+        env.cr.execute(
+            """
+                UPDATE pattern_config SET {} FROM ir_exports
```

Comment:
I prefers to avoid adding logic here

```
                 UPDATE pattern_config
                 SET 
                         export_format=ir_exports.export_format,
                         csv_value_delimiter = ir_exports.csv_value_delimiter,
                         csv_quote_character = ir_exports.csv_quote_character
                 FROM ir_exports
                 WHERE export_id = ir_exports.id
```

It's faster to read and less line of code ;)